### ISSUE TITLE
Call to undefined method get

### ISSUE THREAD

**Issue by twd3** [silentzco/statamic-charge#12]
Copied and pasted subscription create form and script from docs and getting:
Call to undefined method get.C:_statamic\ma\vendor\statamic\cms\src\Tags\Tags.php#127
...when trying to view the template on the front.
```
{{ subscription:create id="the-form" }}
    <input type="text" name="plan" value="price_1HPaqgGOTdH">	
	<input type="text" name="fname">	
	<input type="text" name="lname">

    {{ if errors }}
        Errors:
        <ul>
            {{ errors }}
                <li>{{ value }}</li>
            {{ /errors }}
        </ul>
    {{ /if }}

    {{ if success }}
        Charge successful!
    {{ /if }}

    <!-- Stripe Elements Placeholder -->
    <div id="card-element"></div>

    <button id="card-button" data-secret="{{ subscription:setup_intent }}">
        Subscribe
    </button>
{{ /subscription:create }}	
	
<script>
const stripe = Stripe('{{ config:cashier:key }}');

const elements = stripe.elements();
const cardElement = elements.create('card');

cardElement.mount('#card-element');

const cardButton = document.getElementById('card-button');
const clientSecret = cardButton.dataset.secret;
const form = document.getElementById('the-form');

cardButton.addEventListener('click', async (e) => {
    e.preventDefault();
    cardButton.disabled= true;
    const { setupIntent, error } = await stripe.confirmCardSetup(
        clientSecret, {
            payment_method: {
                card: cardElement,
                billing_details: {
                    name: '{{ fname }} {{ lname }}',
                    email: {{ email }}
                }
            }
        }
    );

    if (error) {
        alert(error);
    } else {
        addToForm('payment_method', setupIntent.payment_method, form);
        form.submit();
    }
});

function addToForm(name, value, form) {
    let input = document.createElement('input');

    input.type = 'hidden';
    input.name = name;
    input.value = value;

    form.appendChild(input);
}
</script>
```

**Comment by edalzell** [comment]
Can you please show me a few more lines of that stack trace?

**Comment by twd3** [comment]
BadMethodCallException
Call to undefined method get.
http://ma.test/become-a-member

        76
        C:\_statamic\ma\resources\views/join.antlers.html
        :0
        75
        C:\_statamic\ma\vendor\statamic\cms\src\Tags\Tags.php
        Statamic\Tags\Tags
        :127
        74
        Silentz\Charge\Tags\BaseTag
        :56

1 unknown frame

    72
    C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Support\Collection.php
    Illuminate\Support\Collection
    :638
    71
    Silentz\Charge\Tags\BaseTag
    :59
    70
    Silentz\Charge\Tags\BaseTag
    :50
    69
    Silentz\Charge\Tags\BaseTag
    :19
    68
    Silentz\Charge\Tags\Subscription
    :21

1 unknown frame

    66
    C:\_statamic\ma\vendor\statamic\cms\src\View\Antlers\Engine.php
    Statamic\View\Antlers\Engine
    :152

    1 unknown frame
        64
        C:\_statamic\ma\vendor\statamic\cms\src\View\Antlers\Parser.php
        Statamic\View\Antlers\Parser
        :575
        63
        Statamic\View\Antlers\Parser
        :176
        62
        Statamic\View\Antlers\Parser
        :131
        61
        Statamic\View\Antlers\Engine
        :89
        60
        Illuminate\View\View
        :139
        59
        Illuminate\View\View
        :122
        58
        Illuminate\View\View
        :91
        57
        Statamic\View\View
        :74
        56
        Statamic\Http\Responses\DataResponse
        :144
        55
        Statamic\Http\Responses\DataResponse
        :45
        54
        Statamic\Structures\Page
        :287
        53
        Illuminate\Routing\Router
        :744
        52
        Illuminate\Routing\Router
        :731
        51
        Illuminate\Routing\Router
        :685
        50
        Illuminate\Pipeline\Pipeline
        :128
        49
        Statamic\StaticCaching\Middleware\Cache
        :29
        48
        Illuminate\Pipeline\Pipeline
        :167
        47
        Statamic\Http\Middleware\Localize
        :22
        46
        Illuminate\Pipeline\Pipeline
        :167
        45
        Illuminate\Routing\Middleware\SubstituteBindings
        :41
        44
        Illuminate\Pipeline\Pipeline
        :167
        43
        Illuminate\Foundation\Http\Middleware\VerifyCsrfToken
        :77
        42
        Illuminate\Pipeline\Pipeline
        :167
        41
        Illuminate\View\Middleware\ShareErrorsFromSession
        :49
        40
        Illuminate\Pipeline\Pipeline
        :167
        39
        Illuminate\Session\Middleware\StartSession
        :116
        38
        Illuminate\Session\Middleware\StartSession
        :62
        37
        Illuminate\Pipeline\Pipeline
        :167
        36
        Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse
        :37
        35
        Illuminate\Pipeline\Pipeline
        :167
        34
        Illuminate\Cookie\Middleware\EncryptCookies
        :67
        33
        Illuminate\Pipeline\Pipeline
        :167
        32
        Statamic\StaticCaching\Middleware\Retrieve
        :34
        31
        Illuminate\Pipeline\Pipeline
        :167
        30
        Illuminate\Pipeline\Pipeline
        :103
        29
        Illuminate\Routing\Router
        :687
        28
        Illuminate\Routing\Router
        :662
        27
        Illuminate\Routing\Router
        :628
        26
        Illuminate\Routing\Router
        :617
        25
        Illuminate\Foundation\Http\Kernel
        :165
        24
        Illuminate\Pipeline\Pipeline
        :128
        23
        Statamic\Http\Middleware\CheckMultisite
        :14
        22
        Illuminate\Pipeline\Pipeline
        :167
        21
        Statamic\Http\Middleware\PoweredByHeader
        :19
        20
        Illuminate\Pipeline\Pipeline
        :167
        19
        Barryvdh\Debugbar\Middleware\InjectDebugbar
        :65
        18
        Illuminate\Pipeline\Pipeline
        :167
        17
        Illuminate\Foundation\Http\Middleware\TransformsRequest
        :21
        16
        Illuminate\Pipeline\Pipeline
        :167
        15
        Illuminate\Foundation\Http\Middleware\TransformsRequest
        :21
        14
        Illuminate\Pipeline\Pipeline
        :167
        13
        Illuminate\Foundation\Http\Middleware\ValidatePostSize
        :27
        12
        Illuminate\Pipeline\Pipeline
        :167
        11
        Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode
        :63
        10
        Illuminate\Pipeline\Pipeline
        :167
        9
        Fruitcake\Cors\HandleCors
        :37
        8
        Illuminate\Pipeline\Pipeline
        :167
        7
        Fideloper\Proxy\TrustProxies
        :57
        6
        Illuminate\Pipeline\Pipeline
        :167
        5
        Illuminate\Pipeline\Pipeline
        :103
        4
        Illuminate\Foundation\Http\Kernel
        :140
        3
        Illuminate\Foundation\Http\Kernel
        :109
        2
        :55
        1
        :143

C:\_statamic\ma\resources\views/join.antlers.html

<div class="w-full bg-black mx-auto">

<div class="max-w-14xl mx-auto border-l border-r border-dotted border-white">

<div class="max-w-screen-xl mx-auto border-l border-r border-dotted border-white">

<div class="max-w-6xl mx-auto font-sans text-white border-l border-r border-dotted border-white">

<div class="h-64 px-2 md:px-4">

<div class="text-center">

<h1 class="text-white">Some big ass page title</h1>



</div>

 

 

 

</div>

</div>

</div>

 

</div>

</div>

 

 



<div class="w-full bg-black mx-auto top-0 sticky z-50">

<div class="max-w-14xl mx-auto border-l border-r border-dotted border-white">

	<div class="max-w-screen-xl mx-auto border-l border-r border-dotted border-white">

		<div class="max-w-6xl mx-auto font-sans text-white border-l border-r border-dotted border-white">

			{{ partial:header }}

		</div>

            

7.4.6977ms30MBGET {segments?}

    Call to undefined method get.C:\_statamic\ma\resources\views/join.antlers.html#0Facade\Ignition\Exceptions\ViewException
    Call to undefined method get.C:\_statamic\ma\vendor\statamic\cms\src\Tags\Tags.php#127BadMethodCallException

        public function __call($method, $args)
        {
            if ($this->wildcardHandled || ! method_exists($this, $this->wildcardMethod)) {
                throw new \BadMethodCallException("Call to undefined method {$method}.");
            }

            $this->wildcardHandled = true;

    #0 C:\_statamic\ma\vendor\silentz\charge\src\Tags\BaseTag.php(56): Statamic\Tags\Tags->__call('get', Array)
    #1 [internal function]: Silentz\Charge\Tags\BaseTag->Silentz\Charge\Tags\{closure}('redirect', 0)
    #2 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Support\Collection.php(638): array_map(Object(Closure), Array, Array)
    #3 C:\_statamic\ma\vendor\silentz\charge\src\Tags\BaseTag.php(59): Illuminate\Support\Collection->map(Object(Closure))
    #4 C:\_statamic\ma\vendor\silentz\charge\src\Tags\BaseTag.php(50): Silentz\Charge\Tags\BaseTag->params()
    #5 C:\_statamic\ma\vendor\silentz\charge\src\Tags\BaseTag.php(19): Silentz\Charge\Tags\BaseTag->hideParams()
    #6 C:\_statamic\ma\vendor\silentz\charge\src\Tags\Subscription.php(21): Silentz\Charge\Tags\BaseTag->createForm('http://major-ve...')
    #7 [internal function]: Silentz\Charge\Tags\Subscription->create()
    #8 C:\_statamic\ma\vendor\statamic\cms\src\View\Antlers\Engine.php(152): call_user_func(Array)
    #9 [internal function]: Statamic\View\Antlers\Engine::renderTag(Object(Statamic\View\Antlers\Parser), 'subscription', Array, '\r\n <input ty...', Array)
    #10 C:\_statamic\ma\vendor\statamic\cms\src\View\Antlers\Parser.php(575): call_user_func_array(Array, Array)
    #11 C:\_statamic\ma\vendor\statamic\cms\src\View\Antlers\Parser.php(176): Statamic\View\Antlers\Parser->parseCallbackTags('<div class="w-f...', Array)
    #12 C:\_statamic\ma\vendor\statamic\cms\src\View\Antlers\Parser.php(131): Statamic\View\Antlers\Parser->parse('<div class="w-f...', Array)
    #13 C:\_statamic\ma\vendor\statamic\cms\src\View\Antlers\Engine.php(89): Statamic\View\Antlers\Parser->parseView('C:\\_statamic\\ma...', '<div class="w-f...', Array)
    #14 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\View\View.php(139): Statamic\View\Antlers\Engine->get('C:\\_statamic\\ma...', Array)
    #15 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\View\View.php(122): Illuminate\View\View->getContents()
    #16 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\View\View.php(91): Illuminate\View\View->renderContents()
    #17 C:\_statamic\ma\vendor\statamic\cms\src\View\View.php(74): Illuminate\View\View->render()
    #18 C:\_statamic\ma\vendor\statamic\cms\src\Http\Responses\DataResponse.php(144): Statamic\View\View->render()
    #19 C:\_statamic\ma\vendor\statamic\cms\src\Http\Responses\DataResponse.php(45): Statamic\Http\Responses\DataResponse->contents()
    #20 C:\_statamic\ma\vendor\statamic\cms\src\Structures\Page.php(287): Statamic\Http\Responses\DataResponse->toResponse(Object(Illuminate\Http\Request))
    #21 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Routing\Router.php(744): Statamic\Structures\Page->toResponse(Object(Illuminate\Http\Request))
    #22 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Routing\Router.php(731): Illuminate\Routing\Router::toResponse(Object(Illuminate\Http\Request), Object(Statamic\Structures\Page))
    #23 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Routing\Router.php(685): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Statamic\Structures\Page))
    #24 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(128): Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
    #25 C:\_statamic\ma\vendor\statamic\cms\src\StaticCaching\Middleware\Cache.php(29): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #26 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Statamic\StaticCaching\Middleware\Cache->handle(Object(Illuminate\Http\Request), Object(Closure))
    #27 C:\_statamic\ma\vendor\statamic\cms\src\Http\Middleware\Localize.php(22): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #28 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Statamic\Http\Middleware\Localize->handle(Object(Illuminate\Http\Request), Object(Closure))
    #29 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Routing\Middleware\SubstituteBindings.php(41): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #30 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Routing\Middleware\SubstituteBindings->handle(Object(Illuminate\Http\Request), Object(Closure))
    #31 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(77): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #32 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
    #33 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #34 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
    #35 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(116): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #36 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(62): Illuminate\Session\Middleware\StartSession->handleStatefulRequest(Object(Illuminate\Http\Request), Object(Illuminate\Session\Store), Object(Closure))
    #37 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
    #38 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #39 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
    #40 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(67): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #41 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
    #42 C:\_statamic\ma\vendor\statamic\cms\src\StaticCaching\Middleware\Retrieve.php(34): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #43 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Statamic\StaticCaching\Middleware\Retrieve->handle(Object(Illuminate\Http\Request), Object(Closure))
    #44 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #45 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Routing\Router.php(687): Illuminate\Pipeline\Pipeline->then(Object(Closure))
    #46 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Routing\Router.php(662): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
    #47 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->runRoute(Object(Illuminate\Http\Request), Object(Illuminate\Routing\Route))
    #48 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Routing\Router.php(617): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
    #49 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(165): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
    #50 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(128): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
    #51 C:\_statamic\ma\vendor\statamic\cms\src\Http\Middleware\CheckMultisite.php(14): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #52 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Statamic\Http\Middleware\CheckMultisite->handle(Object(Illuminate\Http\Request), Object(Closure))
    #53 C:\_statamic\ma\vendor\statamic\cms\src\Http\Middleware\PoweredByHeader.php(19): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #54 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Statamic\Http\Middleware\PoweredByHeader->handle(Object(Illuminate\Http\Request), Object(Closure))
    #55 C:\_statamic\ma\vendor\barryvdh\laravel-debugbar\src\Middleware\InjectDebugbar.php(65): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #56 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Barryvdh\Debugbar\Middleware\InjectDebugbar->handle(Object(Illuminate\Http\Request), Object(Closure))
    #57 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\TransformsRequest.php(21): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #58 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Foundation\Http\Middleware\TransformsRequest->handle(Object(Illuminate\Http\Request), Object(Closure))
    #59 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\TransformsRequest.php(21): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #60 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Foundation\Http\Middleware\TransformsRequest->handle(Object(Illuminate\Http\Request), Object(Closure))
    #61 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\ValidatePostSize.php(27): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #62 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Foundation\Http\Middleware\ValidatePostSize->handle(Object(Illuminate\Http\Request), Object(Closure))
    #63 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(63): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #64 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
    #65 C:\_statamic\ma\vendor\fruitcake\laravel-cors\src\HandleCors.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #66 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Fruitcake\Cors\HandleCors->handle(Object(Illuminate\Http\Request), Object(Closure))
    #67 C:\_statamic\ma\vendor\fideloper\proxy\src\TrustProxies.php(57): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #68 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Fideloper\Proxy\TrustProxies->handle(Object(Illuminate\Http\Request), Object(Closure))
    #69 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
    #70 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(140): Illuminate\Pipeline\Pipeline->then(Object(Closure))
    #71 C:\_statamic\ma\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(109): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
    #72 C:\_statamic\ma\public\index.php(55): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
    #73 C:\Users\Duke\AppData\Roaming\Composer\vendor\cretueusebiu\valet-windows\server.php(143): require('C:\\_statamic\\ma...')
    #74 {main}

**Comment by edalzell** [comment]
If you change `BaseTags.php` line 56 to this:
```
if ($redirect = $this->params->get($param)) {
```

Does that solve your problem?

**Comment by twd3** [comment]
Now:
```
Call to a member function createSetupIntent() on nullC:\_statamic\ma\vendor\silentz\charge\src\Tags\Subscription.php#15Error
```

**Comment by edalzell** [comment]
Ah, if you go into `app/User.php` and change the top so it looks like this:
```
use Silentz\Charge\Chargeable;

class User extends Model
{
    use Notifiable;
    use Chargeable;
```

I bet that fixes it, right?

**Comment by twd3** [comment]
Okay, this is where I was confused a day ago. class User extends Authenticatable was already there so when I added class User extends Model it said User already exists so I just put the use Chargeable; under User extends Authenticatable like:
```
<?php
namespace App;

use Illuminate\Contracts\Auth\MustVerifyEmail;
use Illuminate\Foundation\Auth\User as Authenticatable;
use Illuminate\Notifications\Notifiable;
use Silentz\Charge\Chargeable;

class User extends Authenticatable
{
    use Notifiable;
	use Chargeable;

    protected $fillable = [
        'name', 'email', 'password',
    ];

    protected $hidden = [
        'password', 'remember_token',
    ];

    protected $casts = [
        'email_verified_at' => 'datetime',
    ];
}
```
That other stuff was there so I must need it...what should I do?

**Comment by twd3** [comment]
The default looks like:
```
use Illuminate\Contracts\Auth\MustVerifyEmail;
use Illuminate\Foundation\Auth\User as Authenticatable;
use Illuminate\Notifications\Notifiable;

class User extends Authenticatable
{
    use Notifiable;

    protected $fillable = [
        'name', 'email', 'password',
    ];
    protected $hidden = [
        'password', 'remember_token',
    ];
    protected $casts = [
        'email_verified_at' => 'datetime',
    ];
}
```

**Comment by edalzell** [comment]
> Okay, this is where I was confused a day ago. class User extends Authenticatable was already there so when I added class User extends Model it said User already exists so I just put the use Chargeable; under User extends Authenticatable like:

Yup, looks good

**Comment by twd3** [comment]
Okay, so my User.php is right? I was guessing on that one, didn't want to bug you...haha

**Comment by edalzell** [comment]
Yup, Laravel changed their defaults and I didn't update the docs.

**Comment by edalzell** [comment]
fixed in 4.0.4
